<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkin_form</class>
 <widget class="QDialog" name="checkin_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="wechat">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wechat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bank_card">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Debit Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="credit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="other">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Method: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="cash">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>370</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout Date: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contact">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Room No.: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="checkout_date">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2082</year>
     <month>8</month>
     <day>13</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="alipay">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AliPay</string>
   </property>
  </widget>
  <widget class="QComboBox" name="id_type">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mainland ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Passport</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="gender">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID No.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkin Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone No.: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Checkin Form</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="checkin_date">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2082</year>
     <month>8</month>
     <day>13</day>
    </date>
   </property>
  </widget>
  <widget class="QLineEdit" name="room_number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit Status: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="deposit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Paid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unpaid</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>checkin_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
